src/parser/test/ParserTest.cpp

```diff
@@ -950,6 +950,30 @@ TEST(Parser, FetchVertex) {
         auto result = parser.parse(query);
         ASSERT_TRUE(result.ok()) << result.status();
     }
+    {
+        GQLParser parser;
+        std::string query = "FETCH PROP ON person 1, 2, 3 yield person.id";
+        auto result = parser.parse(query);
+        ASSERT_TRUE(result.ok()) << result.status();
+    }
+    {
+        GQLParser parser;
+        std::string query = "yield 1 as id | FETCH PROP ON person $-.id";
+        auto result = parser.parse(query);
+        ASSERT_TRUE(result.ok()) << result.status();
+    }
+    {
+        GQLParser parser;
+        std::string query = "yield 1 as id | FETCH PROP ON person $-.id yield person.id";
+        auto result = parser.parse(query);
+        ASSERT_TRUE(result.ok()) << result.status();
+    }
+    {
+        GQLParser parser;
+        std::string query = "FETCH PROP ON person, another 1, 2, 3";
```

Comment:
I want to know what's the response looks like in this case.  Person and Another maybe have different props.  You join them on vid together?

Reply:
we use the tag as Wide table, simply speaking: one tag hold one value. In this way, we can support large data adding and modify. 

Person and Another tags join together, display default value if one of them is null.